DiceKeys/Views/Navigation/SupportsBackNavigationHeader.swift

```diff
@@ -43,8 +43,13 @@ struct WithNavigationHeader<HeaderContent: View, BodyContent: View>: View {
 //                            .edgesIgnoringSafeArea(.top)
 //                        }
 //            )
-            Spacer()
+            //Removing this. Let the content decide if it needs Spacer()
+//            Spacer()
             self.content()
-        }.edgesIgnoringSafeArea(.all)}
+        }.edgesIgnoringSafeArea(.leading)
+        .edgesIgnoringSafeArea(.bottom)
+        .edgesIgnoringSafeArea(.trailing)
```

Comment:
When we ignore top area the navigation bar hides behind the app title

Reply:
In iOS or Mac?

When I launch other iOS apps on my iPad, there's no app title on top.  Is that something that's expected?

Reply:
on MAC and it becomes too large on iOS. We will have to use conditional check here